docs/make.jl

```diff
@@ -122,14 +122,12 @@ Downloads.download(
 )
 @info "Final build done"
 
-if "deploy" in ARGS
-    @info "Deploying docs"
-    deploydocs(;
-        repo = "github.com/JuliaAstro/JuliaAstro.github.io",
-        push_preview = true,
-        branch = "master",
-        devbranch = "multidoc-auto-highlevels",
-        cname = "juliaastro.org",
-    )
-    @info "Deploy complete"
-end
+@info "Deploying docs"
```

Comment:
I guess you can drop the `deploy` arg from here now: https://github.com/JuliaAstro/JuliaAstro.github.io/blob/db089cce94ff5f9ea7b67b0b55018975785314c4/.github/workflows/Documentation.yml#L27

Just a note that this will go back to throwing

```julia
┌ Warning: Documenter could not auto-detect the building environment. Skipping deployment.
└ @ Documenter ~/.julia/packages/Documenter/ZlqFj/src/deployconfig.jl:76
```

when building locally

Reply:
xref: https://github.com/JuliaAstro/JuliaAstro.github.io/pull/44#discussion_r2056790921